### ISSUE TITLE
only support ONNX model with IR_VERSION=3

### ISSUE THREAD

**Issue by virgile-thermo** [Microsoft/onnxjs#89]
I converted a keras model using onnxmltools.convert_keras with target_opset=7 but the IR_VERSION is set to 4. 
Is there a way to have compatiblility between the export and your lib?

Best regards,

**Comment by fs-eire** [comment]
ONNX IR_VERSION 4 is published on 2019-01-22 and is not yet supported by ONNX.js now.

I will take a look at the changes of the protobuf definition to see if it is safe to upgrade to IR v4.

**Comment by virgile-thermo** [comment]
thank you so much.

**Comment by GalAster** [comment]
@virgile-thermo You can force `version` to 3.

```py
import onnx
path='test.onnx'
model = onnx.load(path)
model.ir_version = 3
onnx.save(model, path)
```

but if there are some V4 only operators, it will cause `TypeError: unrecognized operator 'XXX'`

**Comment by virgile-thermo** [comment]
Thanks, 
And I's possible do it at the export step : 
onnx_model = onnxmltools.convert_keras(classifier, target_opset=7) 
onnx_model.ir_version = 3
onnxmltools.utils.save_model(onnx_model, 'model.onnx')

**Comment by fs-eire** [comment]
The IR_VERSION support should have been fixed in v0.1.5.

**Comment by hariharans29** [comment]
Closing this as IR v4 is supported now. Please open another issue if you have problems using it.
